app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -351,21 +355,30 @@ export default {
       }
     },
     onFileUpload(file) {
-      this.attachedFiles = [];
-      if (!file) {
-        return;
+      if (file) {
+        if (checkFileSizeLimit(file, MAXIMUM_FILE_UPLOAD_SIZE)) {
+          this.attachedFiles = [];
+          if (!file) {
```

Comment:
This check is not required if you are checking the existence of the file at the beginning of the function.